Add some basic usage docs (#35)
giantswarm/test-infra: docs/create-tekton-task.md

> # Creating a Tekton tasks

Reviewer: ```suggestion
# Creating a Tekton task
```

Tasks are the smallest piece of our test infra puzzle.
They are meant for individual conformance test suite executions
which are supplied either from upstream or internal teams.
This is where you can check the created cluster for conformity
and get the different `kubeconfig` for example as inputs.

All tasks have to be in the `tekton/tasks` folder.

You can discover the upstream docs on Tekton tasks here:
https://tekton.dev/docs/pipelines/tasks/

> An example can look like this:

Reviewer: ```suggestion
An example task might look like this:
```

```yaml
apiVersion: tekton.dev/v1beta1
kind: Task
metadata:
  name: cis # Task name which can be chosen freely.
  namespace: nightly # This namespace should stay unchanged.
spec:
  workspaces:

>   - name: cluster # Workspace supplied be the pipeline, keep naming.

Reviewer: ```suggestion
  - name: cluster # Workspace supplied by the pipeline. Must use the same name.
```
